infracost/1.0.0/artifacthub-pkg.yml

```diff
@@ -0,0 +1,27 @@
+# Artifact Hub package metadata file
+# https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 1.0.0
+name: infracost
+displayName: Infracost Integration
+createdAt: 2022-07-23T00:00:00Z
+description: Keptn integration with Infracost
+logoPath: assets/infracost.png

```

Comment:
Relative paths are currently not supported (see [here](https://github.com/keptn-contrib/artifacthub#adding-images-to-the-readme-or-artifacthub-pkgyml-file))

Reply:
Fixed. I hate this image path issue. Gets me every time!